en/index.md

```diff
@@ -407,6 +407,51 @@ rather than plain English. Try to keep it simple.
 #21 No double space after a period/full stop
 -->
 
+## Admonitions
+
+Admonitions (also commonly referred to as "admonishments" or sometimes
+"callouts" or "Notifications") are a device used in documentation to draw attention to a
+particular statement or paragraph of text. Typically their use is to highlight:
+
+ - A consequence of performing a particular action
+ - An additional source of information which is useful but not required
+ - A helpful tip that will save effort/time
+ - A reminder of some pre-requisite or restriction
+
+ While this style guide usually concerns itself solely with the intent and
+ structure of content rather than the published form of that content, there is
+ a particular intersect here. The [Vanilla framework][], which is used to
+ publish most of our documentation, only allows for a particular set of
+ pre-defined admonitions (or "Notifications" as they are referred to in that
+ framework): **Information** and **Warning**.
+
+ Note that the style of these admonitions relates solely to the colours and
+ icons used with the notes. It is entirely possible to create an Information
+ admonition and use the label 'Useful tip' or to call a Warning 'Danger of
+ imminent death'.
```

Comment:
Perhaps this could be simplified a little:

> Many frameworks allow the title, colour and icon of any admonition to be customised.
> The warning admonition can be titled 'Danger of imminent death' if you prefer.

Might also be worth adding:

> Another common feature is dropdown admonitions, which are initially hidden to readers but can be revealed on click.
> Dropdowns can be helpful if the information contained is important to a select number of readers but optional for others.